partner_time_to_pay/views/res_partner.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<odoo>
+
```

Comment:
Copyright + license would be great

Reply:
@nikiwaibel  Would you please copy copyright from [these file](https://github.com/OCA/purchase-workflow/blob/11.0/stock_landed_cost_company_percentage/views/res_company_view.xml#L3) and paste it here.

Reply:
@nikul-serpentcs Would you please copy copyright from [these file](https://github.com/OCA/purchase-workflow/blob/11.0/stock_landed_cost_company_percentage/views/res_company_view.xml#L3) and use it here.

Reply:
@alexey-pelykh @bodedra No need to here, In XML file not mandatory.

Reply:
@nikul-serpentcs yes. it's not mandatory but it is good practice to have that information. As @alexey-pelykh notice it empty, so it is good time to improve it.

BDW, I will take care of it.  :wink:

Reply:
@nikul-serpentcs I have submit PR https://github.com/nikul-serpentcs/account-invoice-reporting/pull/1. Please merge it.